### PR TITLE
Initial baseline of the dgl and imdb nx adapter post refactoring.

### PR DESCRIPTION
**Pull request by rajivsam**

### CODE REVIEW

File: adbnx_adapter/adbnx_adapter/arangoDB_networkx_adapter.py [hunk 1]
```diff
@@ -20,7 +20,15 @@ def __init__(self, conn):
             user_name = conn["username"]
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/arangoDB_networkx_adapter.py:20]: Should we have defaults here as well?
localhost:8529 no auth?

File: adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+@author: Rajiv Sambasivan
+"""
+
+
+from adbnx_adapter.arangoDB_networkx_adapter import ArangoDB_Networkx_Adapter
+import networkx as nx
+import torch as th
+import numpy as np
+import dgl
+#from arango import ArangoClient
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py:15]: Please delete if not needed

File: adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            for v in vnames:
+                rndata = np.asarray(node_data[v], dtype = int)
+                v_data = th.from_numpy(rndata)
+                g.nodes[v].data['f'] = v_data
+            
+
+
+
+        return g, labels
+
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py:110]: Trailing blanks

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+"""
+Created on Thu Mar 26 09:51:47 2020
+
+@author: Rajiv Sambasivan
+"""
+
+
+from adbnx_adapter.arangoDB_networkx_adapter import ArangoDB_Networkx_Adapter
+import networkx as nx
+#from arango import ArangoClient
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py:12]: Delete commented code

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+#from adbnx_adapter.custom_http_client import CustomHTTPClient
+
+
+
+class IMDBArangoDB_Networkx_Adapter(ArangoDB_Networkx_Adapter):
+
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py:18]: weird whitespace

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    
+            for k,v in  graph_attributes['edgeCollections'].items():
+                query = "FOR doc in %s " % (k)
+                cspl = [s +':'+ 'doc.' + s for s in v]
+                cspl.append('_id: doc._id')
+                csps = ','.join(cspl)
+                query = query + "RETURN { " + csps + "}"
+
+                cursor = self.db.aql.execute(query)
+                #breakpoint()
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py:48]: delete

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                for doc in cursor:
+                    g.add_edge(doc['_from'], doc['_to'])
+
+        return g
+
+
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py:54]: whitespace

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+                for doc in cursor:
+                    if k == "Users":
+                        bip_key = 0
+                    else:
+                        bip_key = 1
+                    g.add_node(doc['_id'], attr_dict = doc, bipartite = bip_key)
+    
+            for k,v in  graph_attributes['edgeCollections'].items():
+                query = "FOR doc in %s " % (k)
+                cspl = [s +':'+ 'doc.' + s for s in v]
```
Comment (joerg84) [inline adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py:42]: what is the resulting identifier? Shouldn't it be `collection/_key` to match the from/to in edge collections
 ?

File: adbnx_adapter/adbnx_adapter/arangoDB_networkx_adapter.py (hunk 1 above)
Comment (rajivsam) [reply adbnx_adapter/adbnx_adapter/arangoDB_networkx_adapter.py:20]: @joerg84 will check that with the next push.

File: adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py (hunk 2 above)
Comment (rajivsam) [reply adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py]: done

File: adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py (hunk 3 above)
Comment (rajivsam) [reply adbnx_adapter/adbnx_adapter/dgl_arangoDB_networkx_adapter.py]: @joerg84 ran autopep8 in the updated push

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py (hunk 4 above)
Comment (rajivsam) [reply adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py]: @joerg84 done

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py (hunk 5 above)
Comment (rajivsam) [reply adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py:15]: @joerg84 ran autopep8 in the updated push

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py (hunk 6 above)
Comment (rajivsam) [reply adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py]: @joerg84 done

File: adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py (hunk 7 above)
Comment (rajivsam) [reply adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py]: @joerg84 ran autopep8 in the updated push

**Pull request merged**